spec/views/occupation_standards/show_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.describe "occupation_standards/show", type: :view do
+  context "without imports feature flag" do
+    it "when public document, it has link to original document" do
+      allow_any_instance_of(ActionView::TestCase::TestController).to receive(:controller_name).and_return("occupation_standards")
+      allow_any_instance_of(ActionView::TestCase::TestController).to receive(:current_user).and_return(nil)
+      allow(ActiveStorage::Current).to receive(:url_options).and_return(host: "https://www.example.com")
+
+      perform_enqueued_jobs do
+        standards_import = create(:standards_import, :with_files, public_document: true)
+        attachment = standards_import.files.first
+        source_file = attachment.source_file
+        occupation_standard = create(:occupation_standard)
+        create(:data_import, source_file: source_file, occupation_standard: occupation_standard)
+
+        assign :occupation_standard, occupation_standard
+
+        render
+
+        expect(rendered).to have_text "View Original Document"
+      end
+    end
+  end
+
+  context "with imports feature flag" do
+    before do
```

Comment:
@littleforest do we need this to be inside a `before` block? Seems we can move it inside the test and remove the block. Same with `after`

Reply:
Ripping out the feature flag soon, and this will make for less clean up than having to remove it from each separate test.

Reply:
(Actually only one test here, but there were multiple tests in another PR I did recently with same format)